### ISSUE TITLE
Delegating through Explorer fails with "error with payload"

### ISSUE THREAD

**Issue by alexlines** [livepeer/livepeer-monorepo#334]
**Describe the bug**
When trying to delegate LPT to a transcoder using the LivePeer explorer and Metamask, the transaction fails (see full error message below). I have an existing amount of LPT already bonded to the transcoder and am trying to bond an additional amount.

**Expected behavior**
I expect the bond to succeed as has happened in the past.

**To Reproduce**
1. Go to https://explorer.livepeer.org/transcoders
2. Sign into Metamask
3. Choose a transcoder and click the "BOND" button
4. Choose an amount
5. Click "SUBMIT"
6. The error screen pops up and bonding fails

**Device**
 - Device: MacBookPro12,1
 - OS: macOS Mojave 10.14.2 (18C54)
 - Browser: Chrome 
 - Version: Version 72.0.3626.109 (Official Build) (64-bit)
 
**Console logs**
rpc error:  
```
[ethjs-rpc]  error with payload {"id":198996783233,"jsonrpc":"2.0","params":[{"to":"0x511bc4556d823ae99630ae8de28b9b80df90ea2e","from":"0x[ETH_ADDRESS]","gas":"0x200b20","value":"0x0","data":"0xb78d27dc0000000000000000000000000000000000000000000001695ddd51d78d20290000000000000000000000000050d69f8253685999b4c74a67ccb3d240e2a56ed6"}],"method":"eth_estimateGas"} Error: JsonRpcEngine - response has no error or result for request:
{
  "id": 1232909163,
  "jsonrpc": "2.0",
  "params": [
    {
      "to": "0x511bc4556d823ae99630ae8de28b9b80df90ea2e",
      "from": "0x[ETH_ADDRESS]",
      "gas": "0x200b20",
      "value": "0x0",
      "data": "0xb78d27dc0000000000000000000000000000000000000000000001695ddd51d78d20290000000000000000000000000050d69f8253685999b4c74a67ccb3d240e2a56ed6"
    }
  ],
  "method": "eth_estimateGas"
}
```

This is not useful but was the only thing in the javascript console:
```
enhance.js:55 bondAndApprove 0x50d69f8253685999b4c74a67ccb3d240e2a56ed6 YY LPT ZZZZ WEI
enhance.js:56 bonding...
index.js:45 notify {type: "error", id: "bond", title: "Error Bonding", body: "There was a problem bonding your token", className: "exited"}body: "There was a problem bonding your token"className: "exited"id: "bond"title: "Error Bonding"type: "error"__proto__: constructor: ƒ Object()hasOwnProperty: ƒ hasOwnProperty()isPrototypeOf: ƒ isPrototypeOf()propertyIsEnumerable: ƒ propertyIsEnumerable()toLocaleString: ƒ toLocaleString()toString: ƒ toString()valueOf: ƒ valueOf()__defineGetter__: ƒ __defineGetter__()__defineSetter__: ƒ __defineSetter__()__lookupGetter__: ƒ __lookupGetter__()__lookupSetter__: ƒ __lookupSetter__()get __proto__: ƒ __proto__()set __proto__: ƒ __proto__()
```

**Comment by iameli** [comment]
Thanks for the report! We saw this error once internally today in the case where Metamask was configured to use a hardware ledger that wasn't connected. Just wanted to rule that out, first?

**Comment by alexlines** [comment]
Confirmed, there's no ledger involved in this transaction nor connected to or expected by Metamask.

**Comment by iameli** [comment]
Thanks for the clarification. The other time I've seen "response has no error or result for request" is when switching networks between Rinkeby/mainnet — I imagine this might happen when switching accounts in Metamask also. https://github.com/livepeer/livepeerjs/issues/322

I don't think that's directly related — I think the error is just saying "Hey I tried to do the Web3 thing and I couldn't" — just noting for posterity as I keep tracking things down.

**Comment by dob** [comment]
Could this be related to #336 ?

**Comment by alexlines** [comment]
I just tried bonding again and rounded to a whole integer and the bond finally succeeded. It's not definitive proof, and I've certainly had bonds succeed in the past with some decimal portion, but it's another data point. lmk if you prefer I close this or keep it open.

**Comment by iameli** [comment]
@alexlines Just rolled out a potential fix for the earlier failure (#342). I'm going to close this for now, but please reopen if you encounter the problem again!
